### PR TITLE
add documentation for enabling/disabling ICE-TCP

### PR DESCRIPTION
**Pull request by fippo**
more :nail_care:

Looking at WebRTCTransport I don't see a way to specify a port to bind here?

### CODE REVIEW

**Comment by ibc** [comment]
I'm a bit lost. Do you mean "documenting how to enable/disable TCP" within the "mediasoup protocol"?

**Comment by fippo** [comment]
yes, can you stop reading my mind? I've only seen this used in some of the tests and I like documentation.

Should I separate the question about whether its possible to bind a single port from the PR?

**Comment by ibc** [comment]
Will come back to this next week :)

**Comment by fippo** [comment]
:beach_umbrella:

File: MEDIASOUP_PROTOCOL.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Request:
   target: 'peer',
   id: 1111,
   direction: 'send', // 'send'/'recv'.
-  options: {},
+  options: {tcp: true}, // enable or disable ICE-TCP
```
Comment (ibc) [inline MEDIASOUP_PROTOCOL.md]: Please, add a ending dot into the comment line :)

Comment (ibc) [reply MEDIASOUP_PROTOCOL.md]: Also, there are other transport options. In fact, the `options` Object in the `createTransport` request matches the [TransportOptions](https://mediasoup.org/documentation/mediasoup-client/api/#Transport-TransportOptions) Object.

**Pull request closed without merging**